setup.cfg

```
@@ -25,7 +25,7 @@ packages =
     find_namespace:
 include_package_data = True
 install_requires =
-    numpy>=1.26,<2 # >1.26 for python 3.12+
+    numpy>=1.26,<3
```

I think this needs to be pandas 2
```suggestion
    numpy>=2.0.0,<3
```